src/Seo/Metadata.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @copyright Copyright (c) Wizacha
+ * @license Proprietary
+ */
+declare(strict_types=1);
+
+namespace Wizaplace\SDK\Seo;
+
+final class Metadata
+{
+    /** @var null|string */
+    private $metaTitle;
+
+    /** @var null|string */
+    private $metaDescription;
+
+    /** @var null|string */
+    private $metaKeywords;
+
+    /**
+     * Metadata constructor.
+     * @param array $data
+     */
+    public function __construct(array $data)
+    {
+        $this->metaTitle = $data['title'];
+        $this->metaDescription = $data['description'];
+        $this->metaKeywords = $data['keywords'];
+    }
+
+    /**
+     * @param string|null $metaTitle
+     * @return Metadata
+     */
+    public function setMetaTitle(?string $metaTitle): self
```

Comment:
```suggestion
    public function setTitle(?string $metaTitle): self
```

Idem pour les setters et getters